RiotSwiftUI/Modules/LocationSharing/LiveLocationSharingViewer/LiveLocationSharingViewerViewModel.swift

```diff
@@ -210,7 +210,9 @@ class LiveLocationSharingViewerViewModel: LiveLocationSharingViewerViewModelType
         guard let foundUserAnnotation = foundUserAnnotation else {
             return
         }
-        
+        if state.showsUserLocationMode == .follow {
+            state.showsUserLocationMode = .show
+        }
```

Comment:
Can you explain a bit the logic here? 🤔
It looks weird.

Reply:
I only want to change the logic if the current user is being followed, but for the .hide state, I don't want to change it in .show